src/client/main.js

```diff
@@ -104,6 +104,42 @@ const app = new Vue({
     playerVariant(player) {
       return player.connected ? 'success' : 'danger';
     },
+    message(state) {
```

Comment:
I think `state` can be accessed in this function using `this.state`, no need to pass it in